From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tobias Werth <werth@cs.fau.de>
Date: Sun, 26 Apr 2020 22:07:10 +0200
Subject: partial cherry-pick: Fix a bunch of typos.

No functional changes intended.

(partially cherry picked from commit 7fd66c6a510a34b35edaf6c5faca90a5f709afed)
---
 webapp/src/Service/ImportExportService.php | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/webapp/src/Service/ImportExportService.php b/webapp/src/Service/ImportExportService.php
index 6cc33c70d..7234a757e 100644
--- a/webapp/src/Service/ImportExportService.php
+++ b/webapp/src/Service/ImportExportService.php
@@ -655,10 +655,10 @@ class ImportExportService
                 ->setCountry($organizationItem['country']);
             $this->em->flush();
             if ($contest = $this->dj->getCurrentContest()) {
-                $this->eventLogService->log('team_affilation', $teamAffiliation->getAffilid(), $action,
+                $this->eventLogService->log('team_affiliation', $teamAffiliation->getAffilid(), $action,
                                             $contest->getCid());
             }
-            $this->dj->auditlog('team_affilation', $teamAffiliation->getAffilid(), 'replaced',
+            $this->dj->auditlog('team_affiliation', $teamAffiliation->getAffilid(), 'replaced',
                                              'imported from tsv / json');
         }
 
-- 
2.20.1

